cmd/thermal-recorder/config_test.go

```diff
@@ -34,13 +34,17 @@ import (
 	"github.com/TheCacophonyProject/window"
 )
 
+var locationFileName = "test_data/location.yaml"
```

Comment:
This makes an assumption about the working directory when the tests are run. You can do something like this to determine the directory of the current file and then construct the path to `test_data` from there:

```go
import (
    "runtime"
    "path/filepath"
)

var locationFileName string

func init() {
   locationFileName = filepath.Join(thisDir(), "test_data", "location.yaml")
}

func thisDir() string {
    _, filename, _, _ := runtime.Caller(0)
   return filepath.Dir(filename)
}
```

This means it doesn't matter what the current working directory was when `go test` was run.

Reply:
Done.